### DISCUSSION TITLE
Termux on android for remote work with distant

### DISCUSSION THREAD

**Discussion by vanarok** [chipsenkbeil/distant#228] [Q&A] [answered]
I work on a tablet connecting to a remote server and termux, the input delay during such work is unpleasant, I was very glad to learn that such a tool as distant is being developed, but unfortunately I was not able to run it in termux.  I was unable to start the manager daemon without third-party programs (systemd, openrs, etc..l) that are required to connect to a remote distant server.  Someone with the same workflow, how did you solve this?

**Comment by chipsenkbeil** [comment]
Hey there! [Termux isn't officially supported (yet)](https://github.com/chipsenkbeil/distant/issues/163), but a build was produced thanks to https://github.com/termux/termux-packages/pull/15610. How is it failing to start? How are you trying to run it?

It looks like they needed to modify the build as shown in [build.sh](https://github.com/termux/termux-packages/blob/cea58eca30039797e4df6ca52ad5fce1dde43920/packages/distant/build.sh#L6) and [changed a file path as shown in paths.rs.patch](https://github.com/termux/termux-packages/blob/cea58eca30039797e4df6ca52ad5fce1dde43920/packages/distant/src-paths.rs.patch). So you may have to run the manager and client pointing to a specialized socket path.

**Comment by vanarok** [comment]
![Screenshot_2023-09-17-14-11-55-760_com termux-edit](https://github.com/chipsenkbeil/distant/assets/6105977/b3523515-0b1a-4e2f-955c-90d8ab354845)
![Screenshot_2023-09-17-14-14-50-340_com termux-edit](https://github.com/chipsenkbeil/distant/assets/6105977/d556993b-3a80-4492-9dc0-f98212815fcd)

**Comment by chipsenkbeil** [comment, marked as answer]
@vanarok you're trying to install the manager as a service, which probably doesn't make sense on termux. Can you try running it normally?

```bash
# Start the manager as a daemon process
distant manager listen --daemon

# Print out the connections, which should be none by default -- this will fail if unable to connect to the manager
distant manager list
```
